server/src/utils/http_service.rs

```diff
@@ -226,6 +229,11 @@ async fn create_game(state: AsyncState) -> Vec<u8> {
     ..Default::default()
   };
 
+  let game_state = GameStateT {
+      game: Some(Box::new(game)),
```

Comment:
Is there any way to avoid using a `Box` here? I think perhaps `game.clone()` may work, but am unsure.

Reply:
Yeah, I would prefer to keep as much as I could on the stack. I'll check to see if this is possible, but I vaguely remember this being one of the things that the rust compiler recommended as a fix. I am not nearly smart enough to decide to use a `Box` on my own, lmfao.